packages/components/src/components/permissions/actions.ts

```diff
@@ -36,11 +36,7 @@ function processPrincipalsResponse(data: ISelectRowsResult, resolve, isGroup: bo
         principals = principals.push(principal);
     });
 
-    if (isGroup) {
```

Comment:
with the removal of this if, I think you can also remove the unused param `isGroup` as well

Reply:
updated